### PR TITLE
ROC-7643: Add support endpoint for orders reprint

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-7643

### Change description ###

ROC-7643: Add support endpoint for orders reprint

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportControllerTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportController.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5135254&bid=16873381)

**Comment by timja** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportControllerTest.java  1
- src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportController.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/cmc-claim-store/pullRequest?prid=5135254&bid=16873381)

File: src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportController.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        this.caseMapper = caseMapper;
+    }
+
+    @PutMapping("/claim/{referenceNumber}/resend-order-for-print")
+    @ApiOperation("Resend staff notifications associated with provided event")
+    public void resendLegalAdvisorOrderToPrint(@PathVariable("referenceNumber") String referenceNumber) {
+        User user = userService.authenticateAnonymousCaseWorker();
+        String authorisation = user.getAuthorisation();
+
+        Claim claim = claimService.getClaimByReferenceAnonymous(referenceNumber)
```
Comment (karthikkannankainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportController.java]: Have this as first line, if at all there is no reference we can avoid login userService.authenticateAnonymousCaseWoker()

Comment (dharmendrak) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/orders/BulkPrintSupportController.java]: sure, will action in any other PR

**Pull request merged**